### ISSUE TITLE
formatting doesn't work

### ISSUE THREAD

**Issue by ar1a** [emacs-lsp/lsp-vue#4]
Error:
```
Error from the Language Server: Request textDocument/formatting failed with message: Cannot read property ’format’ of undefined (Internal Error)
```

Reproduce steps:

- Head into vue buffer (with code in it)
- M-x lsp-format-buffer

**Comment by lawrsp** [comment]
I also have error: 

```
Error from the Language Server: Request textDocument/formatting failed with message: Cannot read property ’eslintIntegration’ of undefined (Internal Error)
```

**Comment by sivakov512** [comment]
This error has been fixed with #16 
Please re-open this issue if your problem does not solved.
